common/changes/@microsoft/rush/ianc-rebuild_2020-04-11-01-02.json

```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "@microsoft/rush",
+      "comment": "Fix a regression where an overridden \"rebuild\" script previously executed the \"rebuild\" script.",
```

Comment:
`"Fix a regression in Rush 1.2.3 where customizing \"rush rebuild\" would [describe what it did wrong]."`

Heh, I'm having trouble understanding what this changelog was trying to say.